pkg/crds/names.go

```diff
@@ -0,0 +1,300 @@
+package crds
+
+import "github.com/rancher/rancher/pkg/features"
+
+// RequiredCRDs returns a list of CRD to install based on enabled features.
+func RequiredCRDs() []string {
```

Comment:
I'm curious about how these were all found and how they were grouped together. Are we sure we're not missing any? Is this feature-based CRD installation consistent with how things work today (e.g. we don't install `users.management.cattle.io` unless the "auth" feature is enabled?)?